src/plugins/explore/public/components/patterns_table/utils/utils.ts

```diff
@@ -215,8 +215,12 @@ export const findDefaultPatternsField = (services: ExploreServices): string => {
     return field.type === 'string';
   });
 
+  if (!logResults?.hits?.hits?.[0]) {
+    throw new Error('Cannot access hits from logs tab');
+  }
```

Comment:
i'm weary of throwing errors like this as our fallback is pretty bad (and things we typically we want to do when you catch an error, such as log it, is not possible in our system), is there a way not to throw error?

Reply:
Oh so in this function, I throw a lot of errors for unexpected behavior (which was "causing" the original problem), but a step above where this is called, I run a try catch for this function so this error isn't going to be thrown and handled like any other error. 
I'm fine with changing this to however it needs to be, though. Should I remove all the `throw new Errors` I have in my code?

Reply:
we synced offline, its fine as long as we do not get our users to a state where they no longer have access to tabs